### DISCUSSION TITLE
impolite words

### DISCUSSION THREAD

**Discussion by Atefeh197** [openai/whisper#1534] [General]
Hi everyone

I see whisper transcribes impolite words with ***, is there any way to transcribe them with letters?
It decreases WER.

**Comment by phineas-pta** [comment]
whisper transcribe swearwords without censor

**Reply by Atefeh197** [reply]
But I see it transcribes these words with *** for example s***!

**Reply by phineas-pta** [reply]
it transcribe this video without censor
https://youtu.be/kTeQSzHGWyw

**Comment by Atefeh197** [comment]
thank you for your response,
Is there any parameter for that, do you know? 
Because it will be censored.

**Reply by phineas-pta** [reply]
u try transcribe the video i send and got censor ?
